packages/core/src/components/ChoiceList/Select.example.jsx

```diff
@@ -3,14 +3,36 @@ import ReactDOM from 'react-dom';
 import Select from './Select';
 
 ReactDOM.render(
-  <Select defaultValue="2" name="select-demo">
-    <option value="1">Option 1</option>
-    <option value="2">Option 2</option>
-    <option value="3">Option 3</option>
-    <option value="4">Option 4</option>
-    <option value="5">Option 5</option>
-    <option value="6">Option 6</option>
-    <option value="7">Option 7</option>
-  </Select>,
+  <div>
+    <Select defaultValue="2" name="select-demo">
+      <option value="1">Option 1</option>
+      <option value="2">Option 2</option>
+      <option value="3">Option 3</option>
+      <option value="4">Option 4</option>
+      <option value="5">Option 5</option>
+      <option value="6">Option 6</option>
+      <option value="7">Option 7</option>
+    </Select>
+    <h4>Small size modifier</h4>
+    <Select defaultValue="2" size="small" name="small-select-demo">
```

Comment:
Hm, this is actually cutting off the full label. Should we change these options to reflect a "Suffix" list (i.e. `Sr.`, `Jr.`) so the text doesn't get cut off? 

![image](https://user-images.githubusercontent.com/371943/37042068-ac9c5986-212b-11e8-9d68-fbda7d5e4bee.png)